### DISCUSSION TITLE
Can't find Notes property and NotesCollection type.

### DISCUSSION THREAD

**Discussion by Jenix-Park** [melanchall/drywetmidi#183] [Q&A] [answered]
[This](https://melanchall.github.io/drywetmidi) overview page contains an example code like below :
```
using (var notesManager = trackChunk.ManageNotes())
{
    NotesCollection notes = notesManager.Notes;
...
```
But both Notes property and NotesCollection class don't exist.

Also, I have an old project with DryWetMidi in it.
I have a code like the below, but there are no Events and TimedEventsCollection either.
`TimedEventsCollection timedEvents = timedEventManager.Events;`

**Comment by melanchall** [comment, marked as answer]
Hi,

Yes, these classes and properties are deprecated by the latest version of the library: https://melanchall.github.io/drywetmidi/obsolete/obsolete.html#obs11.

So your examples should be transformed to:

```csharp
TimedObjectsManager<Note> notes = notesManager.Objects;
```

or

```csharp
var notes = notesManager.Objects;
```

The same for timed events.

**Reply by Jenix-Park** [reply]
Thanks for the quick reply!
Btw, I found your mistake in the code above.
It's "TimedObjectsCollection".

So, after getting the "TimedObjectsCollection<T>" objects via the Objects property,
are all others the same as before? (eg. Add() method)
I mean, don't I need to worry about breaking changes in my previous code after just switching the old types with the new "TimedObjectsCollection<T>"?

**Reply by melanchall** [reply]
Yes, all methods should remain the same.
